src/forms/set-group-select.schema.js

```diff
@@ -11,7 +11,6 @@ const setGroupSelectSchema = (intl) => ({
   noValueUpdates: true,
   isMulti: true,
   isSearchable: true,
-  simpleValue: false,
```

Comment:
Is it sending the correct data? You might now just get the option value not the whole object.